index.html

```diff
@@ -88,6 +88,10 @@
         <p>Created by <a href="https://twitter.com/_Shakeel" target="_blank">@_Shakeel</a></p>
         <p>Source available on <a href="https://github.com/zen-audio-player/zen-audio-player.github.io" target="_blank">GitHub</a></p>
         <p><a href="https://chrome.google.com/webstore/detail/zen-youtube-audio-player/jlkomkpeedajclllhhfkloddbihmcjlm" target="_blank">Get the Chrome extension</a> - (<a href="https://github.com/zen-audio-player/extension-chrome" target="_blank">source</a>)</p>
+        <!-- Sponsor links -->
+	<br>
+        <p><h3>Sponsored by:</h3></p>
+        <p><a href="https://www.browserstack.com/" target="_blank"><img src="img/browserstack_logo.png" alt="Browserstack logo"></a></p>
```

Comment:
Sorry about that. I'm new to the GitHub editor, and I think I misunderstood you the first time. How does it look now?